Releases 1.0.0

v-soujanya opened the pull request:
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command
<!--- Please provide the related command with az {command} if you can, so that we can quickly route to the related person to review. --->


### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).


### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

yonzhan commented:
<!-- add-pr-comment:prcommentbot -->

Thank you for your contribution! We will review the pull request and get back to you soon.

v-soujanya commented:
Hi @jsntcy as you suggested in earlier PR I changed the devops extension version 0.27.0 to 1.0.0 and added a new entry for this version. Still PR pipeline is failing with same issue as earlier. can you please assist here as it is blocking to publish the release.

jsntcy commented on src/index.json:
```diff
@@ -17133,6 +17133,56 @@
+                    "name": "azure-devops",
+                    "run_requires": [
+                        {
+                            "requires": [
```
If you download https://github.com/Azure/azure-devops-cli-extension/releases/download/20240206.1/azure_devops-1.0.0-py2.py3-none-any.whl and open "\azure_devops-1.0.0-py2.py3-none-any\azure_devops-1.0.0.dist-info\metadata.json", you can find metadata in whl file is different from the metadata in this PR.


![image](https://github.com/Azure/azure-cli-extensions/assets/16348853/84ffcaed-7287-44ec-a23b-e18e52905a0d)

![image](https://github.com/Azure/azure-cli-extensions/assets/16348853/0a34b986-3e67-4d86-9244-0fb38a0f9142)

The CI error message also shows the root cause

![image](https://github.com/Azure/azure-cli-extensions/assets/16348853/aebb7816-c8d9-441c-bbaf-7fe892323528)

jsntcy commented on src/index.json:
```diff
@@ -17133,6 +17133,56 @@
+                    "name": "azure-devops",
+                    "run_requires": [
+                        {
+                            "requires": [
```
If you download https://github.com/Azure/azure-devops-cli-extension/releases/download/20240206.1/azure_devops-1.0.0-py2.py3-none-any.whl and open "\azure_devops-1.0.0-py2.py3-none-any\azure_devops-1.0.0.dist-info\metadata.json", you can find metadata in whl file is different from the metadata in this PR.


![image](https://github.com/Azure/azure-cli-extensions/assets/16348853/84ffcaed-7287-44ec-a23b-e18e52905a0d)

![image](https://github.com/Azure/azure-cli-extensions/assets/16348853/0a34b986-3e67-4d86-9244-0fb38a0f9142)

The CI error message also shows the root cause

![image](https://github.com/Azure/azure-cli-extensions/assets/16348853/aebb7816-c8d9-441c-bbaf-7fe892323528)

v-soujanya replied:
@jsntcy Thanks for all your assistance here, I made the changes as you suggested, and pipeline checks have passed. Please approve this PR. Thanks a lot!

v-soujanya commented:
@jsntcy please approve this PR as it has passed all checks. Thank you!

v-soujanya commented:
@yonzhan @jsntcy Hi Team, can someone please approve this PR? it is blocking us to publish the CLI Devops extension.
